src/core/widget-config.ts

```diff
@@ -39,4 +39,6 @@ function separateProps(
     return { options, defaults, templates };
 }
 
-export { separateProps };
+export {
```

Comment:
Should be default?

Reply:
Nope, the file is "widget-config.ts", it would be strange if `default export` was `separateProps `